### DISCUSSION TITLE
Dark Mode

### DISCUSSION THREAD

**Discussion by IrvingFSanchez** [bionic-gpt/bionic-gpt#404] [Ideas]
I tried searching for Dark mode in the discussions but it was nowhere to be found, with that being said, could we please get an integrated dark mode option for the chat bot. My eyes are blaring charizard flames from looking at websites that are behind the times with their solar atomic white default pages. Yes I know I could use an extension, but the less extensions I use, the better. Thank you good sir's ma'am's 🔥👄🔥

-sincerely, my blazing red eyes

**Comment by ianpurton** [comment]
Click on the button bottom left, the one with your email address in it and select dark mode.

Hope this message gets to you in time to put the fire out.
